crates/storage/src/params.rs

```diff
@@ -70,6 +72,13 @@ impl<Inner: ToSql> ToSql for Option<Inner> {
     }
 }
 
+impl ToSql for TrieStorageIndex {
```

Comment:
maybe use `to_sql_int!`?

Reply:
That would require the type to follow the `Felt`-like `new` and `get` "interface" methods, and I deemed those pointless as there's no need to hide the inner `u64`, hence the custom `impl ToSql` there.

Reply:
Aha, OK.